website_odoo_debranding/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: AGPLv3 License
+
+===================================
+Remove odoo.com bindings on website
```

Comment:
I'd say `Remove Odoo Branding from Website` instead, just so we can cover the more abstract - particularly given the module rename